test/specs/platform.dom.tests.js

```diff
@@ -107,13 +107,13 @@ describe('Platform.dom', function() {
 				}
 			}, {
 				canvas: {
-					style: 'width: 425px'
+					style: 'width: 212px'
```

Comment:
This test needs to verify the same values, defaults need to be changed before and after the test:

```javascript
it('should use default "chart" aspect ratio for render and display sizes', function() {
    var ratio = Chart.defaults.doughnut.aspectRatio;
    Chart.defaults.doughnut.aspectRatio = 1;

    var chart = acquireChart({
        type: 'doughnut',
        // ...

    expect(chart).toBeChartOfSize({
        // ...
    });

    Chart.defaults.doughnut.aspectRatio = ratio;
});
```